cni-plugins.yaml

```diff
@@ -96,6 +100,10 @@ subpackages:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/usr/bin
           cp bin/${{range.key}} "${{targets.subpkgdir}}"/usr/bin
+    test:
+      pipeline:
+        - runs: |
+            ${{range.key}} --version
   - range: plugins
```

Comment:
nit: let's add spaces between these subpackages

Reply:
`yam` removes the space between `subpackages`, so CI fails 😞 
adding the space would definitely help readability though

Reply:
The `.yam.yaml` accounts for a gap between the subpackages: https://github.com/wolfi-dev/os/blob/main/.yam.yaml#L3, which makes this confusing

Reply:
That is incredibly strange, but given everything else is in order, let's get this through